lib/customers.js

```diff
@@ -263,16 +263,20 @@ function getComplianceTypes () {
 function updateOverride (fields) {
   const updateableFields = [
     'id_card_data',
-    'id_card_photo',
-    'front_camera',
+    'id_card_photo_path',
+    'front_camera_path',
     'authorized',
     'us_ssn'
   ]
 
   const updatedFields = _.intersection(updateableFields, _.keys(fields))
-  const atFields = _.fromPairs(_.map(f => [`${f}_override`, 'automatic'], updatedFields))
+  const overrideFields = _.fromPairs(_.map(f =>
+    _.endsWith('_path', f)
+      ? [`${_.replace('_path', '_override', f)}`, 'automatic']
+      : [`${f}_override`, 'automatic']
+    , updatedFields))
```

Comment:
Can't we just remove the _path at the end of every string and get rid of the ternary if here?

Reply:
I took another look at the code base, and I don't think so. I contextualized this fix in more detail on a comment in the respective card, but this problem happens because the DB column for `id_card_photo` and `front_camera` have the `_path` suffix, which caused the strings without the `_path` to not match with any DB column, and not update the `_override` column to `automatic`.

Simply putting it, the issue is because all other columns are structured using the logic `<column_name><suffix>` which leads to something like `id_card_data`, and with the suffix, `id_card_data_override`. With `id_card_photo` and `front_camera`, the respective columns are  `id_card_photo_path` and `id_card_photo_override`. As the string array only has `id_card_photo`, and not `id_card_photo_path`, the `_.intersection()` call does not match with the appropriate DB column, which leads to the `_override` lines to not be executed for these two columns, which leads to the `id_card_photo_override` not being updated to `automatic` and staying in `blocked`, which leads to the infinite loop

Reply:
I just meant something like:

```js
  const atFields = _.fromPairs(_.map(f => {
    const noPath = _.replace(f, '_path$', '');
    return [`${noPath}_override`, 'automatic'], updatedFields)
  })
```

Reply:
To get it even more readable you can probably do something like:

```js
    const removePathSuffix = _.replace('_path$', '')
    const getPairs = _.map(f => [`${removePathSuffix(f)}_override`, 'automatic'])
    const overrideFields = _.fromPairs(getPairs, updatedFields))
```